### PR TITLE
more input checks and test cases

### PR DESCRIPTION
**Pull request by duanyutong**
more input checks and test cases

### CODE REVIEW

File: tests/dracula/test_run_topp_jnt_crt.py [hunk 1]
```diff
@@ -11,36 +12,49 @@
...
     alim_jnt = 7.5 * lim_ones  # rad/s^2
-    waypts_crt = np.loadtxt("test_waypts_crt_0.txt")
-    lim_ones = np.ones(waypts_crt.shape[1])
+    lim_ones = np.ones(waypts_crt_list[0].shape[1])
     lim_ones = np.vstack([-lim_ones, lim_ones]).T
     vlim_crt = 1 * lim_ones  # m/s
     alim_crt = 10 * lim_ones  # m/s^2
 
     def test_run_topp_jnt_crt(self):
         """Test run_topp_jnt_crt(), just assert no throw."""
```
Comment (5yler) [inline tests/dracula/test_run_topp_jnt_crt.py]: Can you add some more detail on this test case. What is being tested at a high level, where does the test data come from, what is the expected output?

File: tests/dracula/test_run_topp.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def test_run_topp_spline_static_data_(self):
 
     def test_run_topp_const_accel_static_data(self):
         """Test run_topp_const_accel() using static test data."""
-        for coeff in [1, 0.5, 0.2, 0.1, 0.05]:
+        for coeff in [1, 0.5, 0.1]:
```
Comment (5yler) [inline tests/dracula/test_run_topp.py:43]: Why the change?

Comment (duanyutong) [reply tests/dracula/test_run_topp.py:43]: 0.05 * panda's limits is too low, even lower than the epsilon so it's not supported. took 0.2 out to speed up unit test

File: tests/dracula/test_run_topp_jnt_crt.py (hunk 1 above)
Comment (duanyutong) [reply tests/dracula/test_run_topp_jnt_crt.py]: yes, added more description to docstring in a493a07

File: tests/dracula/test_run_topp_jnt_crt.py [hunk 3]
```diff
@@ -11,36 +12,55 @@
...
+                self.alim_crt,
+                return_cs=True,
+            )
+            v_abs = np.abs(cs.derivative(1)(cs.x))
+            a_abs = np.abs(cs.derivative(2)(cs.x))
+            self.assertAlmostEqual(v_abs[0].max(), 0)
+            self.assertAlmostEqual(v_abs[-1].max(), 0)
+            self.assertAlmostEqual(a_abs[0].max(), 0)
+            self.assertAlmostEqual(a_abs[-1].max(), 0)
+            self.assertGreater(cs.x[-1], 0.479)
```
Comment (gavincangan) [inline tests/dracula/test_run_topp_jnt_crt.py:63]: Shouldn't this be greater than or equal?

Comment (duanyutong) [reply tests/dracula/test_run_topp_jnt_crt.py:63]: no because for this particular known test case the duration is greater than that after taking into account cartesian limits but smaller without them

**Review by gavincangan — APPROVED** [review]
